astra/src/test/java/com/slack/astra/elasticsearchApi/ElasticsearchApiServiceTest.java

```diff
@@ -137,7 +137,7 @@ public void testSchemaFields() throws Exception {
         objectMapper.convertValue(
             jsonNode.get("test").get("mappings").get("properties"), Map.class);
     assertThat(map).isNotNull();
-    assertThat(map.size()).isEqualTo(31);
+    assertThat(map.size()).isEqualTo(25);
```

Comment:
These are the 6 pre-defined fields that won't show up in the mapping API.
hostname
duration_ms
package
payload
tag
type


Previously they would have shown up even if no one had ever added a record for it. So this is a good change